jax/_src/random.py

```diff
@@ -417,11 +417,10 @@ def uniform(key: ArrayLike,
     raise ValueError(f"dtype argument to `uniform` must be a float dtype, "
                      f"got {dtype}")
   dtype = dtypes.canonicalize_dtype(dtype)
-  if out_sharding is None:
+  def f(key, minval, maxval, shape, dtype):   # reorder args
```

Comment:
You can get rid of this `f` entirely and reorder the args in `_uniform` to `_uniform(key, minval, maxval, shape, dtype)`

See https://critique.corp.google.com/cl/743340725/depot/google3/third_party/py/jax/_src/random.py